ci/pipeline

```diff
@@ -189,32 +171,6 @@ def buildAndTestDockerAndLinuxPackages(): Unit = {
   }
 }
 
-/**
```

Comment:
We can introduce it again if it is needed. However, as I understand the tags will be defined by the build version.

Reply:
Is the plan for us to create tags for versions that we deem as stable?

Reply:
As I understood we only tag community releases. A community release was soaked for two weeks and thus is considered stable. @meichstedt and @kensipe what are your takes on this?

Reply:
@jeschkies tagging community releases seems sufficient to me. As stated in the scope doc
> Promoting the above build to a community release means tagging the related commit [...] and publishing relevant artifacts [...].

Depending on how often/how many builds we soak, we could tag every build that has been soaked and is considered stable; Once every 4-6 weeks should be enough though, so we don't end up with too many versions.

(In a later improvement I'd like to integrate with JIRA and not only tag these commits, but also create the according versions in JIRA).